File: server-scala-play-server/app/model/VehicleType.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for VehicleType.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class VehicleType(
  id: Option[Long],
  active: Option[Boolean],
  valid: Option[Boolean],
  business: Option[Retailer],
  hub: Option[ServiceHub],
  name: Option[String],
  width: Option[Double],
  height: Option[Double],
  depth: Option[Double],
  volume: Option[Double],
  maxWeight: Option[Double],
  vehicleCargoSettings: Option[List[VehicleCargoSetting]]
)

object VehicleType {
  implicit lazy val vehicleTypeJsonFormat: Format[VehicleType] = Json.format[VehicleType]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/VehicleType.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class VehicleType 
{
    public long Id { get; set; }
    public bool Active { get; set; }
    public bool Valid { get; set; }
    public Retailer Business { get; set; }
    public ServiceHub Hub { get; set; }
    public string Name { get; set; }
    public double Width { get; set; }
    public double Height { get; set; }
    public double Depth { get; set; }
    public double Volume { get; set; }
    public double MaxWeight { get; set; }
    public List<VehicleCargoSetting> VehicleCargoSettings { get; set; }
}


